cardano-node/src/Cardano/Node/Query.hs

```diff
@@ -0,0 +1,191 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+-- | Answer queries using the ledger state of the running node
+--
+-- This allows queries from within the running node as opposed to submitting
+-- queries via the LocalStateQuery protocol.
+module Cardano.Node.Query
```

Comment:
This module does not seem to be used, or I missed it.

I would not expect it to be used, not for extending the CLI command.

Reply:
Removed!